kong/pdk/log.lua

```diff
@@ -640,7 +641,12 @@ do
 
   local function is_valid_value(v, visited)
     local t = type(v)
-    if v == nil or t == "number" or t == "string" or t == "boolean" then
+
+    -- cdata is not supported by cjson.encode
+    if type(v) == 'cdata' then
```

Comment:
Should we check `userdata` too? but notice that `ngx.null` is the only userdata which  can be encoded.